src/document/DocumentCommandHandlers.js

```diff
@@ -1462,9 +1462,28 @@ define(function (require, exports, module) {
         }
     }
 
-    /** Navigate to the next/previous (MRU) document. Don't update MRU order yet */
-    function goNextPrevDoc(inc) {
-        var result = MainViewManager.traverseToNextViewByMRU(inc);
+    /**
+     * Finds the next/previous document in MRU or list order
+     * @param {!number} inc Delta indicating in which direction we're going
+     * @param {?boolean} listOrder Whether to navigate using MRU or list order. Defaults to MRU order
+     * @return {?{file:File, paneId:string}} The File object of the next item in the travesal order or null if there aren't any files to traverse.
+     *                                       May return current file if there are no other files to traverse.
+     */
+    function findNextPrevDoc(inc, listOrder) {
```

Comment:
Do we need this function? It feels like those 4 lines can be added to `goNextPrevDoc` ?